src/Grid.js

```diff
@@ -1,21 +1,18 @@
 import React, { Component } from 'react';
 import Button from './button';
+import Step from './Step'
```

Comment:
moved `Step` to it's own component to handle the logic...but realizing this may not be the best route.